app/views/posts/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="post-info">
+    <div class="post-user-info">
+      <p>Post <%= '#' %><%= @post.id %><%= ' ' %><%= @post.title %> by <%= @post.author.name %></p>
+      <p>Comments: <%= @post.comments_counter || 0 %> 
+      <span> Likes: <%= @post.likes_counter || 0 %></span></p>
+    </div>
+    <p><%= @post.text %></p>
+    <div class="comments">
+         <% @post.recent_comments.each do |comment| %>
+            <p><%= comment.author.name %> : <%= comment.text %> </p>
+        <% end %>
+    </div>
```

Comment:
- [ ] Good job using `post.recent_comments` to render 5 most recent comments, however, this should be in the `app/views/posts/index.html.erb` page rather. Kindly make changes and display all comments here.

Reply:
Thanks cwaku, the information was quite helpful.